cdn/webpack.config.js

```diff
@@ -163,8 +186,46 @@ const config = (target) => {
           }
         }
       ]
-    }
+    },
+    output: {
+      ...commonConfig.output,
+      ...{
+        // Because the ./agent-aggregator/aggregator.js dependency is async loaded, the output filename for that chunk will be the
+        // same across all bundles in non-production builds (where filenames aren't hashed) and will thus overwrite with either an
+        // ES5 or ES6 target. For differentiated transpilation of dynamically loaded dependencies in non-production builds, we can
+        // tag output filenames for chunks of the polyfills bundle with `-es5`.
+        chunkFilename: SUBVERSION === 'PROD' ? `[name].[hash:8]${PATH_VERSION}.js` : `[name]-es5${PATH_VERSION}.js`,
```

Comment:
[suggestion] It might actually be good to always have a tag like `-polyfill` on the polyfill files, even for production.